README.md

```diff
@@ -29,26 +29,34 @@ To get it running end-to-end locally with mocked dataset, you will need to set u
 
 Just follow the README of each of them for instructions on how to get it to run.
 
-### How to run tests in debug mode
+### Remote debugging with VS Code
```

Comment:
it is worth adding a sentence that debugging remotely works also for staging and production urls and that for the debugger to work, the developer has to have the same version for the code of the ui locally, as it is for the environment they are trying to debug

Reply:
Fixed.